app/templates/layouts/_base.html

```diff
@@ -1,5 +1,5 @@
 <!doctype html>
-{% set cdn_hash = "v1.8.4" %}
+{% set cdn_hash = "eae2f7d" %}
```

Comment:
This is while we are waiting on the release on PL right?. We moved away from using the hash to release tags for visibility.

Reply:
Yeah once we create PL v1.8.5 I'll change this